egs/callhome_diarization/v1/diarization/vad_to_segments.sh

```diff
@@ -58,7 +58,9 @@ if [ $stage -le 0 ]; then
 
   for n in `seq $nj`; do
     cat $sdata/$n/subsegments
-  done | sort > $data/subsegments || exit 1;
+  done | sort | \
+  awk '{if (! (NF != 4 || $4 - $3 <= 0.25)) { print $0 }}' \
```

Comment:
sure, but I think it would make more sense to have the minimum length configurable as an option to the script. 
You can pass it into awk using e.g. -v m=$min_duration
Also I prefer if you write that as an && expression instead of a ! ||.

Reply:
Sorry the conditional was bad indeed. Fixed it and passed the duration as an option.